packages/python/plotly/plotly/tests/test_core/test_px/test_px_hover.py

```diff
@@ -151,3 +151,15 @@ def test_fail_wrong_column():
         "Ambiguous input: values for 'c' appear both in hover_data and data_frame"
         in str(err_msg.value)
     )
+
+
+def test_sunburst_hoverdict_color():
```

Comment:
this test fails on master because it tries to append to a dict

Reply:
👍